benchmark/benchmark.cpp

```diff
@@ -423,7 +420,7 @@ void Benchmark<T>::get_avail_chunks(
         for (i = 0; i < k; i++) {
             int j = c_chunks_id->at(i);
             avail_c_chunks->at(j) = a_streams->at(j);
-            avail_c_props->at(j) = c_propos->at(j);
+            avail_c_props.at(j) = c_propos.at(j);
```

Comment:
is c_propos a typo ? should be c_props ?

Reply:
Yes, could be.
Typo seems also present in `fec_utest.cpp` (`propos_flags`), I'll fix it in a dedicated commit (but in this PR)

Reply:
Yes, it is :)